internal/tool/ssh/deploy.go

```diff
@@ -87,12 +87,14 @@ func deploy(ctx context.Context, args []string) error {
 	}
 
 	// Copy the binaries to each location.
-	if err := copyBinaries(locs, dep); err != nil {
+	tmpDirs, err := copyBinaries(locs, dep)
+
```

Comment:
remove empty line before checking the error. here and everywhere else

we typically do:
bla, err := ...
if err != nil {
 ....
}